content/en/docs/collector/trace-receiver.md

```diff
@@ -234,30 +218,23 @@ requirements above.
 ```go
 package tailtracer
 
-import (
-  "go.opentelemetry.io/collector/config"
-)
-
 // Config represents the receiver config settings within the collector's config.yaml
 type Config struct {
-   config.ReceiverSettings `mapstructure:",squash"`
    Interval    string `mapstructure:"interval"`
    NumberOfTraces int `mapstructure:"number_of_traces"`
 }
 ```
 
 > #### Check your work
 >
-> - I imported the `go.opentelemetry.io/collector/config` package, which is
->   where ReceiverSettings is declared.
-> - I embedded the `config.ReceiverSettings` as required by the spec.
 > - I added the `Interval` and the `NumberOfTraces` fields so I can properly
 >   have access to their values from the config.yaml.
 
 Now that you have access to the settings, you can provide any kind of validation
 needed for those values by implementing the `Validate` method according to the
-[validatable](https://github.com/open-telemetry/opentelemetry-collector/blob/v{{%param
-collectorVersion %}}/config/common.go#L24) interface.
+optional
+[ConfigValidator](https://github.com/open-telemetry/opentelemetry-collector/blob/v0.69.0/component/config.go#L50)
```

Comment:
Do not hard code collector versions, instead use the version variable like this (make this change everywhere in the document):

```suggestion
[ConfigValidator](https://github.com/open-telemetry/opentelemetry-collector/blob/{{% param collectorVersion %}}/component/config.go#L50)
```

Reply:
Actually, since I'm in the file now, let me make those changes.

Reply:
Done